ao_crm_helpdesk/views/res_partner_view.xml

```diff
@@ -15,6 +15,13 @@
                     <field name="helpdesk_count" widget="statinfo" string="Helpdesk"/>
                 </button>
             </div>
+            <page name='sales_purchases' position="after">
+                <page name='helpdesk' string="Helpdesk">
+                    <group>
+                        <field name="helpdesk_default_priority"/>
```

Comment:
add `widget="priority"`

Reply:
forget this too :laughing:  can be misleading as you do not have a way to distinguish between not set and 0 with the widget.